squeezed/src/squeezed.ml

```diff
@@ -106,6 +106,8 @@ let bind () =
   S.get_domain_zero_policy get_domain_zero_policy
 
 let _ =
+  Debug.set_facility Syslog.Local5 ;
+  Debug.set_level Syslog.Info ;
```

Comment:
I think that almost everything is logged as `debug`, so this may be a bit harsh. The facility is the main one to get things out of daemon.log. But what about the logging to stdout?

Reply:
I've removed the custom logging facilities, it still could be better by changing the "verbose" variables passed around.